generated/thrift/rpc.thrift

```diff
@@ -92,6 +93,22 @@ struct WriteRequest {
 	3: required Datapoint datapoint
 }
 
+struct FetchMetadataBatchRawRequest {
+	1: required binary nameSpace
+	2: required list<binary> ids
+}
+
+struct FetchMetadataBatchRawResult {
+	1: required list<FetchMetadataRawResult> elements
+}
+
+struct FetchMetadataRawResult {
```

Comment:
this struct doesn't have a field to indicate which series it's talking about, do you need that?

Reply:
I added an `id` field here. But I took a look at the `FetchBatchRaw` endpoint which accepts a list of ids and returns a list of segments which are in the same order as the list of requested ids. I dont personally love this pattern but if we want to maintain this pattern for consistency that is fine.

Reply:
Yeah to save space (and allocs, run time, etc) we do not return ID's, simply the order in which the data was requested.  The raw and batch endpoints aren't meant to be as most efficient as can possibly be.

Reply:
I will remove this then.Thanks for the tip rob.